public/app/features/alerting/unified/components/rules/CloneRuleButton.tsx

```diff
@@ -20,10 +19,10 @@ export const CloneRuleButton = React.forwardRef<HTMLAnchorElement, CloneRuleButt
   ({ text, ruleIdentifier, isProvisioned, className }, ref) => {
     // For provisioned rules an additional confirmation step is required
     // Users have to be aware that the cloned rule will NOT be marked as provisioned
-    const [provRuleCloneUrl, setProvRuleCloneUrl] = useState<string | undefined>(undefined);
+    const [showModal, setShowModal] = useState(false);
```

Comment:
I'm not sure why the state was previously tracking the same value as `cloneUrl`, it's always computed when the component is rendered and we were always using a boolean check for this value anyway.

Reply:
Me neither 😅  The change looks good!